integration/query_set_test.go

```diff
@@ -0,0 +1,287 @@
+// Copyright 2021 FerretDB Set.
```

Comment:
That should be in update_test.go or at update_something.go. query_XXX is for query tests

Reply:
renamed.